app/cafe.py

```diff
@@ -0,0 +1,18 @@
+from app.errors import NotVaccinatedError,\
+    OutdatedVaccineError, NotWearingMaskError

```

Comment:
Use ( if you want import, try to avoid '\' everywhere
```suggestion
from app.errors import (
    NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
)

```